compiler/grainformat/walktree.re

```diff
@@ -148,61 +156,105 @@ let walktree =
   // );
 };
 
+let rec partition_comments_internal =
```

Comment:
Would maybe like to see a doc comment on either this or `partition_comments`, just so it's clear what knobs these parameters are turning.

Reply:
Great idea, that's in!